UI/multiview.cpp

```diff
@@ -74,6 +74,8 @@ void Multiview::Update(MultiviewLayout multiviewLayout, bool drawLabel,
 	multiviewScenes.clear();
 	multiviewLabels.clear();
 
+	RefreshColors();
```

Comment:
> `Update` is the method that renders the multiview. There is no need to fetch these colors every single frame.
> 
> Set up a signal to handle this update if necessary.

@Warchamp7 Multiview has a seperate Render method that is called every single frame to render the Multiview.

The 'Update' method that has a call to the new utility method 'RefreshColors' was inserted into is only called once when the Multiview projector is opened, and anytime a multiview relevant setting is changed, such as the layout or these new accessibility colors. It is not called every frame, and may even only be called once during the lifetime of the OBS processes.

Please correct me if I am wrong in this, but between breakpoints and extra logging calls inside both the 'Update' and 'Render' methods, and code inspection of calls to the 'Update' method and the Multiview class both not inheriting any other class, or being inherited by any other class leading to no hidden virtual function calls, leads me to believe this is the case.

The 'Update' method was originally used to calculate various offset constants that are specific to the multiview layout setting, as this was the only setting that multiview even had. I added the call to 'RefreshColors' to this specifically because it is not called very often, the 'RefreshColors' method is computationally trivial in this context, it is already called upon multiview projector creation so there is less initialization code to add elsewhere, and the existing hierarchy of calls that eventually invokes this method does not have to be modified either.

I can setup a signal for this if still necessary, if you could direct me to the relevant place where this should be setup, as none of the other accessibility colors settings use signals and I have no examples to go off from them.

The RefreshColors method and call of it inside the Update method was created in this manner to as closely resemble existing accessibility color usage in their respective contexts.